docs/.gitignore

```diff
@@ -14,10 +14,11 @@ versions.json
 
 # Generated API docs.
 # Note: keep `mdx` files that are handwritten
-docs/api/*
-!docs/api/**/*.mdx
-versioned_docs/*/api/*
-!versioned_docs/*/api/**/*.mdx
+docs/api
+!docs/api/index.mdx
+versioned_docs/*/api
+!versioned_docs/*/api/index.mdx
```

Comment:
Opted to exclude specific files, rather than blanket-excluding `mdx` files. This will let us (theoretically) move our API docs to `mdx` in the future.